src/CardTheme.res

```diff
@@ -128,6 +128,8 @@ let getVariables = (str, dict, default, logger) => {
       "spacingGridColumn",
       "spacingGridRow",
       "spacingAccordionItem",
+      "sdkHandleConfirmPayment",
+      "sdkHandleOneClickConfirmPayment",
```

Comment:
Bro these are not appearance variables right.. let keep only appearance variable here